velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -274,6 +274,14 @@ variant variantAt<TypeKind::TIMESTAMP>(
       dataChunk->GetValue(column, row).GetValue<::duckdb::timestamp_t>()));
 }
 
+template <>
+variant variantAt<TypeKind::UNKNOWN>(
```

Comment:
When would this be called? I believe there is a check for null before this call. If that's the case, then, perhaps, a safer implementation would be to throw VELOX_UNREACHABLE.

Reply:
Just removed it and will cover it in another PR.